src/Cake.Common/Tools/WiX/Heat/HeatSettings.cs

```diff
@@ -0,0 +1,252 @@
+using System.Collections.Generic;
```

Comment:
@RLittlesII can you add the new license file header? You will find an example on all the other files in the repo. This is a new requirement after joining the .Net Foundation.